test/e2e/cross_os/test_worker_status.py

```diff
@@ -47,19 +47,19 @@ def is_worker_started() -> bool:
 
         assert is_worker_started()
 
-        deadline_client.update_worker(
-            farmId=deadline_resources.farm.id,
-            fleetId=deadline_resources.fleet.id,
-            workerId=function_worker.worker_id,
-            status=WorkerStatus.STOPPED,
+        function_worker.send_command(
```

Comment:
There's a helper function for this. You shouldn't need to craft the ssm commands directly.

[Windows](https://github.com/aws-deadline/deadline-cloud-test-fixtures/blob/afbc5fb88f981976d60635cfb84d409eda65ffb8/src/deadline_test_fixtures/deadline/worker.py#L489-L493) & [Linux](https://github.com/aws-deadline/deadline-cloud-test-fixtures/blob/afbc5fb88f981976d60635cfb84d409eda65ffb8/src/deadline_test_fixtures/deadline/worker.py#L691-L695)

Please use those instead. That way the test doesn't need updating if there are ev. then the test doesn't n

Reply:
Refactored and published the change!